dev/php.ini

```diff
@@ -1,17 +1,28 @@
 [PHP]
-; Prod settings -- see ../prod/php.ini
+; Prod settings
 date.timezone = UTC
-zend.assertions	= 0
-upload_max_filesize = 24M
-post_max_size = 32M
 file_uploads = On
-
-; CI settings - see ../ci/php.ini
-; TODO Change memory_limit to prod value.
-memory_limit = 1024M
-max_execution_time = 0
-zend.assertions = 1
+display_errors = Off
+display_startup_errors = Off
+max_execution_time = 120
+max_input_time = 900
+max_input_vars = 10000
+opcache.enable = 1
+opcache.enable_cli = 1
+opcache.interned_strings_buffer = 16
+opcache.max_accelerated_files = 65407
+opcache.max_file_size = 0
+opcache.memory_consumption = 128
+opcache.revalidate_freq = 0
+opcache.validate_timestamps = Off
+post_max_size = 100M
+realpath_cache_size = 800K
+realpath_cache_ttl = 86400
+session.cookie_lifetime = 2000000
+session.gc_maxlifetime = 2000000
+upload_max_filesize = 100M
 
 ; Dev settings
```

Comment:
We should probably still layer the CI changes on top of the prod changes and under the dev changes as we've done previously.

Reply:
I don't understand, the changes are still included :-D